tdrs-backend/tdpservice/parsers/test/test_validators.py

```diff
@@ -1,6 +1,8 @@
 """Tests for generic validator functions."""
 
+import pytest
 from .. import validators
+from tdpservice.parsers.test.factories import TanfT5Factory
```

Comment:
We make a T4 factory but don't need to use it? Maybe I missed it in test_parse?

Reply:
Hmm, I hesitate to include unused/untested code in a PR. Let's remove it for now but ensure to not delete this branch when merging in case we want to reference it later.